process-migration-service/src/test/resources/META-INF/persistence.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <persistence
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    version="2.1"
-    xmlns="http://xmlns.jcp.org/xml/ns/persistence"
-    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
-    <persistence-unit name="migration-test" transaction-type="RESOURCE_LOCAL">
-        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
-        <class>org.kie.processmigration.model.Plan</class>
-        <class>org.kie.processmigration.model.Migration</class>
-        <class>org.kie.processmigration.model.MigrationReport</class>
-        <properties>
-            <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect" />
-             <!-- Configuring JDBC properties -->
-            <property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test"/>
-            <property name="javax.persistence.jdbc.driver" value="org.h2.Driver"/>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  version="2.1"
+  xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
+  <persistence-unit name="migration-test" transaction-type="RESOURCE_LOCAL">
+    <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+    <class>org.kie.processmigration.model.Plan</class>
+    <class>org.kie.processmigration.model.Migration</class>
+    <class>org.kie.processmigration.model.MigrationReport</class>
+    <properties>
+      <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
+      <!-- Configuring JDBC properties -->
+      <property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test"/>
+      <property name="javax.persistence.jdbc.driver" value="org.h2.Driver"/>
```

Comment:
I noticed that we rely on Hibernate to create actual datasource for us. Maybe it would be better to use proper JTA for tests by creating a datasource and only then creating the Persistence Context. See [LocalTaskAuditTest](https://github.com/kiegroup/jbpm/blob/217fea90a419db1f0d8aecdc85883caa0085b840/jbpm-human-task/jbpm-human-task-audit/src/test/java/org/jbpm/services/task/audit/service/LocalTaskAuditTest.java#L38) in jbpm as an example.